core/domain/fs_domain.py

```diff
@@ -20,7 +20,6 @@
 import os
 
 import cloudstorage
```

Comment:
cloudstorage is third-party. You can manually check these things, btw. If Oppia doesn't have a cloudstorage.py file then it's not part of Oppia.

Reply:
I will include them on the third party list. Done!